src/app/workspaces/create-workspace/select-stack/ready-to-go-stacks/ready-to-go-stacks.controller.js

```diff
@@ -25,22 +25,37 @@ class ReadyToGoStacksCtrl {
     this.$rootScope = $rootScope;
 
     this.stacks = [];
+    this.stack = null;
 
     let promiseStack = codenvyStack.fetchStacks();
     promiseStack.then(() => {
+        this.updateData();
+      },
+      (error) => {
+        // etag handling so also retrieve last data that were fetched before
+        if (error.status === 304) {
+          // ok
           this.updateData();
-        },
-        (error) => {
-          // etag handling so also retrieve last data that were fetched before
-          if (error.status === 304) {
-            // ok
-            this.updateData();
-          }
-        });
+        }
+      });
 
+  }
+
+  //TODO should change cdvy-simple-selecter widget
```

Comment:
I guess we should renounce the practice with a practice of using callback controller in widget, because it isn't clear what callback controller should to implement without code review. In this case we should write to many comments and keep them up to date. And for now the simple select widget has one function in comment "cdvySimpleSelecter(name)" but can call two " cdvySimpleSelecter(name, value) and cdvySimpleSelecterDefault(value) ".  By the way, we haven't necessity for using callback controller in this small widget.  WDYT?